DevoxxRemoteFunctions/build.gradle

```diff
@@ -25,6 +47,11 @@ task conferenceZip(type: Zip) {
     }
 }
 
+task updateConferenceLambda(type: AWSLambdaUpdateFunctionCodeTask, dependsOn: conferenceZip) {
+    functionName = "devoxxConference"
+    zipFile = new File("$buildDir/distributions/conference.zip")
```

Comment:
You can use the property of the Zip task directly instead of hardcoding the path:

    zipFile = conferenceZip.archiveFile